cmd/executor/cmd/root.go

```diff
@@ -272,6 +282,21 @@ func checkContained() bool {
 	return proc.GetContainerRuntime(0, 0) != proc.RuntimeNotFound
 }
 
+// checkNoDeprecatedFlags return an error if deprecated flags are used.
```

Comment:
This would break users of the existing flags, and wouldn't be effectively different than just removing the old flag immediately, albeit with a nicer error message.

I think we should allow the deprecated flag and just log a warning instead of failing.

Reply:
👍